src/object/authenticate.h

```diff
@@ -235,7 +235,8 @@ extern int au_change_owner (MOP class_mop, MOP owner_mop);
 extern int au_change_class_owner (MOP class_mop, MOP owner_mop);
 extern int au_change_serial_owner (MOP serial_mop, MOP owner_mop, bool by_class_owner_change);
 extern int au_change_trigger_owner (MOP trigger_mop, MOP owner_mop);
-extern int au_change_sp_owner (MOP sp, MOP owner);
+extern int au_change_sp_owner (PARSER_CONTEXT * parser, MOP sp, MOP owner);
+extern int au_change_sp_owner_pre (PARSER_CONTEXT * parser, MOP sp_mop, MOP owner_mop);
```

Comment:
```suggestion
extern int au_change_sp_owner_internal (PARSER_CONTEXT * parser, MOP sp, MOP owner);
extern int au_change_sp_owner (PARSER_CONTEXT * parser, MOP sp_mop, MOP owner_mop);
```

함수명에 대한 저의 의견은 다음과 같은데, 다른 분들의 의견이 궁금합니다.
- au_change_owner를 au_change_owner_internal로 변경
- au_change_sp_owner_pre를 au_change_sp_owner로 변경
- au_change_sp_owner를 au_change_sp_owner_internal로 변경

위와 같이 수정하면 콜스택을 다음과 같습니다.
- au_change_class_owner() -> au_change_class_owner_internal()
- au_change_sp_owner() -> au_change_sp_owner_internal() 

위와 같이 생각하는 이유는 sp_owner 라는 개념이 생기면서 au_change_owner가 의미하는 것이 모호해졌기 때문입니다.(au_change_owner의 의미가 class인지 sp인지 구분 불가)

만약 au_change_owner의 수정사항이 큰 영향을 미친다면 이번에는 sp만 수정하고 추후에 변경하는 것도 좋다고 생각합니다.

참고 사항으로 아래 그림을 보시면 wrapper의 경우 함수명()->함수명_internal()로 작명하는 것을 볼 수 있습니다.
![image](https://github.com/user-attachments/assets/4811af59-90d5-4eea-bb01-d46e9b29e4a1)

Reply:
다음은 어떨까요? 함수 이름에서 명확히 어떤 것을 수행하는지 알 수 있을 것 같습니다.
* au_change_owner -> au_change_class_owner_with_partitions
* au_change_class_owner (유지)
* au_change_sp_owner (유지)

Reply:
@kangmin5505 @hgryoo 

코드 리뷰 감사합니다.
Description 의 Implementation 에 관련 내용을 수정 했습니다.

코드 반영 : to-do (반영 후 삭제 예정)

#### history 기록을 위해 작성 합니다.
* `*_internal` 함수의 접미사는 CUBRID Convention에 따라 extern이 아닌 static으로 불려야 함